lib/administration/ticket/include/irods/ticket_administration.hpp

```diff
@@ -0,0 +1,577 @@
+                                                 constraint.value,
+                                                 "",
+                                                 false);
+            }
+            else if constexpr (std::is_same_v<group_constraint, TicketConstraint>) {
+                detail::execute_ticket_operation(conn,
+                                                 "mod",
+                                                 ticket_name,
+                                                 "remove",
+                                                 "group",
+                                                 constraint.value,
+                                                 "",
+                                                 false);
+            }
+            else if constexpr (std::is_same_v<host_constraint, TicketConstraint>) {
+                detail::execute_ticket_operation(conn,
+                                                 "mod",
+                                                 ticket_name,
+                                                 "remove",
+                                                 "host",
+                                                 constraint.value,
+                                                 "",
+                                                 false);
+            }
+
+            else if constexpr (std::is_same_v<use_count_constraint, TicketConstraint>) {
+                detail::execute_ticket_operation(conn, "mod", ticket_name, "uses", "0", "", "", false);
+            }
+            else if constexpr (std::is_same_v<n_writes_to_data_object_constraint, TicketConstraint>) {
+                detail::execute_ticket_operation(conn, "mod", ticket_name, "write-file", "0", "", "", false);
+            }
+            else if constexpr (std::is_same_v<n_write_bytes_constraint, TicketConstraint>) {
+                detail::execute_ticket_operation(conn, "mod", ticket_name, "write-bytes", "0", "", "", false);
+            }
+            else {
+                throw std::invalid_argument("Invalid constraint type.");
+            }
+        }
+
+        /// \brief Remove the ticket constraint from the ticket
+        ///
+        /// \tparam TicketConstraint The template struct to indicate the type of constraint
+        /// \param[in] conn The communication object
+        /// \param[in] ticket_id The id for the ticket
+        /// \param[in] constraint The constraint type that should be removed from the ticket
+        ///
+        /// \since 4.3.1
+        template <typename TicketConstraint>
+        void remove_ticket_constraint(RxComm& conn, const int ticket_id, const TicketConstraint& constraint)
+        {
+            remove_ticket_constraint(conn, std::to_string(ticket_id), constraint);
+        }
+
+        /// \brief Remove the ticket constraint from the ticket using admin privilege
+        ///
+        /// \tparam TicketConstraint The template struct to indicate the type of constraint
```

Comment:
Add an empty line after this line so that the `\param` directive is more noticeable.